esp/src/src-react/util/history.ts

```diff
@@ -41,6 +42,15 @@ export function parseSearch<T = ParsedQuery<string | boolean | number>>(_: strin
     return { ...parse(_.substring(1), { parseBooleans: true, parseNumbers: true }) } as unknown as T;
 }
 
+export function parseSort(filter: ParsedQuery<string | boolean | number>): QuerySortItem {
+    if (filter?.sortBy) {
+        const sortBy = filter.sortBy.toString().split(":");
+        return { attribute: sortBy[0], descending: sortBy[1] === "DESC" ? true : false };
+    } else {
+        return { attribute: undefined, descending: false };
+    }
+}
```

Comment:
This logic should be rolled into "parseSearch" which in turn should return `{filter:{...}, sort?:{column:string, descending:boolean}}`